src/library/Uno.Themes.Markup.WinUI/Theme.Brushes.cs

```diff
@@ -0,0 +1,435 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.MarkupHelpers;
+using Microsoft.UI.Xaml.Media;
+
+namespace Uno.Themes.Markup
+{
+	partial class Theme
+	{
+		public static class Brushes
```

Comment:
Not sure what needs to be done here, this could be used by anyone that is building their app with C# Markup. You're saying the Figma plugin wouldn't be able to convert a `Backgound="{StaticResource PrimaryBrush}"` to `.Background(Theme.Brushes.Primary.Default);` ?

Reply:
I mean this would be something we should communicate with the figma plugin team.
Non-flat strucuture is fine, so long as they are able to distinguish base brushes (having to add Default suffix) vs their opacity variants.

Reply:
Ah I see, yeah good point. Been discussing a little with Carl, generation from Figma may not end up using this for the time being, but something to keep track of for sure